titus-api/src/main/java/com/netflix/titus/api/jobmanager/model/job/sanitizer/JobConfiguration.java

```diff
@@ -63,14 +63,14 @@
     @DefaultValue("524288")
```

Comment:
You probably want to reduce this as well?

Reply:
I can do this in another PR.